### PR TITLE
Enh/5503

### PR DESCRIPTION
**Pull request by vsemradu**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [ ] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [ ] It's submitted to the `develop` branch, _not_ the `master` branch if no hotfix
- [ ] When resolving a specific issue, it's referenced in the PR's description (e.g. `Fix #xxx[,#xxx]`, where "xxx" is the Github issue number)
- [ ] All tests are passing
- [ ] New/updated tests are included
- [ ] Changelog was modified

If adding a **new feature**, the PR's description includes:
- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

### CODE REVIEW

File: protected/humhub/modules/user/models/ProfileField.php [hunk 1]
```diff
@@ -310,7 +311,13 @@ public function checkType()
      */
     public function getUserValue(User $user, $raw = true): ?string
     {
-        return $this->fieldType->getUserValue($user, $raw);
+        $data = $this->fieldType->getUserValue($user, $raw);
+
+        if (!empty($this->fieldType->prefix)){
+            $data =Html::a($data, $this->fieldType->prefix.$data);
```
Comment (ArchBlood) [inline protected/humhub/modules/user/models/ProfileField.php:317]: Missing spaces it looks like between `prefix` and `.` as well as `$data` and `Html` and `=`

File: protected/humhub/modules/user/models/fieldtype/Text.php [hunk 2]
```diff
@@ -70,6 +70,13 @@ class Text extends BaseType
      */
     public $canBeDirectoryFilter = true;
 
+    /**
+     * Prefix - tel://, sip://,xmpp:// etc
+     *
+     * @var String
+     */
+    public $prefix;
```
Comment (luke-) [inline protected/humhub/modules/user/models/fieldtype/Text.php:78]: Can you please add  @since 1.11

Comment (luke-) [reply protected/humhub/modules/user/models/fieldtype/Text.php:78]: Please rename to `$linkPrefix`. Then it's clearer this also creates a link.

**Review by luke- — CHANGES_REQUESTED** [review]
@vsemradu Thanks. Please also check the Code Style notes from @ArchBlood

**Pull request merged**